### PR TITLE
Fix LanguageBind models normalization parameters parse

### PR DESCRIPTION
**Pull request by wanliAlex**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
bug fix

* **What is the current behavior?** (You can also link to an open issue here)
we are having an issue if we call vectorise on a language bind model. we see:
```
    on_start(_config)
  File "/app/src/marqo/tensor_search/on_start_script.py", line 45, in on_start
    thing_to_start.run()
  File "/app/src/marqo/tensor_search/on_start_script.py", line 180, in run
    _ = _preload_model(model=model, content=test_string, device=device)
  File "/app/src/marqo/tensor_search/on_start_script.py", line 278, in _preload_model
    _ = vectorise(
  File "/app/src/marqo/s2_inference/s2_inference.py", line 72, in vectorise
    return _vectorise_without_cache(model_cache_key, content, normalize_embeddings, modality, media_download_headers,
  File "/app/src/marqo/s2_inference/s2_inference.py", line 135, in _vectorise_without_cache
    return _encode_without_cache(model_cache_key, content, normalize_embeddings, modality, media_download_headers, **kwargs)
  File "/app/src/marqo/s2_inference/s2_inference.py", line 146, in _encode_without_cache
    vectorised = model.encode(
  File "/app/src/marqo/s2_inference/multimodal_model_load.py", line 116, in encode
    print(**kwargs)
TypeError: 'normalize' is an invalid keyword argument for print()
```

* **What is the new behavior (if this is a feature change)?**
we fix the bug

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Have unit tests been run against this PR?** (Has there also been any additional testing?)

no
* **Related Python client changes** (link commit/PR here)
no

* **Related documentation changes** (link commit/PR here)
no

* **Other information**:
no

* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Review by RaynorChavez — DISMISSED** [review]

File: tests/s2_inference/test_large_model_encoding.py [hunk 1]
```diff
@@ -367,10 +369,46 @@ def test_multilingual_e5_model_performance(self):
...
+
+    def help_test_vectorise(self, model_name):
+        for content in ["test", ["test2", "test3"]]:
+            normalized_embeddings_list = vectorise(model_name=model_name,
+                                                   content=content, device="cuda", normalize_embeddings=True)
+            for embeddings in normalized_embeddings_list:
+                self.assertTrue(np.linalg.norm(np.array(embeddings)) - 1 < 1e-6)
+
+            unnormalized_embeddings_list = vectorise(model_name=model_name,
+                                                   content=content, device="cuda", normalize_embeddings=True)
```
Comment (papa99do) [inline tests/s2_inference/test_large_model_encoding.py]: normalize_embeddings=False?

Comment (wanliAlex) [reply tests/s2_inference/test_large_model_encoding.py]: Thanks! fixed.

File: tests/s2_inference/test_large_model_encoding.py [hunk 2]
```diff
@@ -367,10 +369,46 @@ def test_multilingual_e5_model_performance(self):
...
+    @classmethod
+    def tearDownClass(cls) -> None:
+        clear_loaded_models()
+        remove_cached_model_files()
+
+    def setUp(self):
+        self.models = ["LanguageBind/Video_V1.5_FT_Audio_FT_Image"]
+
+    def help_test_vectorise(self, model_name):
+        for content in ["test", ["test2", "test3"]]:
```
Comment (farshidz) [inline tests/s2_inference/test_large_model_encoding.py]: Need to test all supported modalities

File: tests/s2_inference/test_large_model_encoding.py (hunk 1 above)
Comment (papa99do) [reply tests/s2_inference/test_large_model_encoding.py]: Did the test case pass without this change? If so, the test is not capturing any issue.

**Review by farshidz — DISMISSED** [review]

File: tests/s2_inference/test_large_model_encoding.py (hunk 1 above)
Comment (wanliAlex) [reply tests/s2_inference/test_large_model_encoding.py]: We have updated tests.

File: tests/s2_inference/test_large_model_encoding.py (hunk 2 above)
Comment (wanliAlex) [reply tests/s2_inference/test_large_model_encoding.py]: Added tests

**Review by farshidz — APPROVED** [review]

**Pull request merged**